packages/mobile/src/screens/edit-track-screen/screens/CoverAttributionScreen.tsx

```diff
@@ -0,0 +1,74 @@
+import { useField } from 'formik'
+
+import { Box, Divider, Flex } from '@audius/harmony-native'
+import { Switch, Text } from 'app/components/core'
+import { HarmonyTextField } from 'app/components/fields'
+import { FormScreen } from 'app/screens/form-screen'
+
+const messages = {
+  title: 'Cover Attribution',
+  isCover: 'This Song is a Cover',
+  description: 'This track was written by another artist.',
+  attribution: {
+    header: 'Please provide the original song title and artist name.',
+    originalSongTitle: 'Original Song Title',
+    originalArtist: 'Original Artist'
+  }
+}
+
+export const COVER_ORIGINAL_SONG_TITLE = 'cover_original_song_title'
+export const COVER_ORIGINAL_ARTIST = 'cover_original_artist'
+export const IS_COVER = 'isCover'
```

Comment:
should maybe match snake case of other field names (use `_`)

Reply:
ah ur probly right.. but in general should try to match existing patterns

Reply:
seems to be the exisiting pattern for `FormValues` used for submission in `EditTrackScreen` for the values that aren't sent to BE to be in camelCase